Synview-backend/websocket/websocket_server.ts

```diff
@@ -0,0 +1,21 @@
+import { PostQuestion, Question } from "../../common/types.ts";
+
+const sockets = new Set<WebSocket>();
+
+export async function EntrySocket(socket: WebSocket): Promise<void> {
+  socket.onopen = () => sockets.add(socket);
+  socket.onclose = () => sockets.delete(socket);
+  socket.onerror = () => sockets.delete(socket);
+  return;
+}
+
+export function sendQuestionToConnectedUsers(data: PostQuestion) {
+  const question = JSON.stringify(data);
+  for (const socket of sockets) {
+    try {
+      socket.send(question);
+    } catch {
+      sockets.delete(socket);
```

Comment:
Could you send some kind of error to the client first before deleting the WS? 

Also - mentioned this before but it is important - would strongly recommend creating a Logger wrapper and logging errors in situations like this, otherwise, when things randomly start working it becomes very difficult to understand what happened. I would definitely log the exception message especially

Reply:
edit: nvm, ignore, it probably makes sense that you cannot send an error to the client in this case